### PR TITLE
Add short namespace snippet (.NET 6)

### PR DESCRIPTION
**Pull request by MRmlik12**
When C# 10 released, new cleaner way to implement `namespace` was [added](https://docs.microsoft.com/pl-pl/dotnet/csharp/fundamentals/types/namespaces). To activate this snippet type `snamespace` and press enter.

### CODE REVIEW

File: snippets/csharp.json [hunk 1]
```diff
@@ -302,6 +302,13 @@
         ],
         "description": "Namespace"
     },
+    "ShortNamespace": {
+        "prefix": "snamespace",
+        "body": [
+            "namespace ${1:Name};"
+        ],
+        "description": "Short namespace (.NET 6)"
```
Comment (JoeRobich) [inline snippets/csharp.json:310]:
```suggestion
        "description": "File-scoped namespace"
```

File: snippets/csharp.json [hunk 2]
```diff
@@ -302,6 +302,13 @@
         ],
         "description": "Namespace"
     },
+    "ShortNamespace": {
+        "prefix": "snamespace",
```
Comment (JoeRobich) [inline snippets/csharp.json:306]: I think we should go with either "fsnamespace" or "fsn". What do you think?

**Review by JoeRobich — APPROVED** [review]
@MRmlik12 Thanks for contributing this snippet. Just a few suggestions.

File: snippets/csharp.json (hunk 2 above)
Comment (MRmlik12) [reply snippets/csharp.json:306]: I think, it was great idea. I change it

**Pull request merged**